.github/workflows/ci_workflows.yml

```diff
@@ -0,0 +1,135 @@
+            os: ubuntu-latest
+            python: 3.7
+            toxenv: py37-test-alldeps-numpy117-cov-clocale
+            toxposargs: --remote-data=astropy
+
+          - name: Python 3.8 with all optional dependencies (Windows)
+            os: windows-latest
+            python: 3.8
+            toxenv: py38-test-alldeps
+
+          - name: Python 3.7 with remote data and dev version of key dependencies
+            os: ubuntu-latest
+            python: 3.7
+            toxenv: py37-test-devdeps
+            toxposargs: --remote-data=any
+
+          # Cron jobs
+
+          - name: Python 3.7 with all optional dependencies (MacOS X)
+            cron: true
+            os: macos-latest
+            python: 3.7
+            toxenv: py37-test-devdeps
+            toxposargs: --remote-data=astropy
+            # compiler: clang
+
+          # We check numpy-dev also in a job that only runs from cron, so that
+          # we can spot issues sooner. We do not use remote data here, since
+          # that gives too many false positives due to URL timeouts. We also
+          # install all dependencies via pip here so we pick up the latest
+          # releases.
+          - name: Python 3.7 with dev version of key dependencies
+            skip: ${{ github.event_name != 'schedule'}}
+            os: ubuntu-latest
+            python: 3.7
+            toxenv: py37-test-devdeps
+
+          - name: Documentation link check
+            skip: ${{ github.event_name != 'schedule'}}
+            os: ubuntu-latest
+            python: 3.7
+            toxenv: linkcheck
+
+          - name: Bundling with pyinstaller
+            skip: ${{ github.event_name != 'schedule'}}
+            os: ubuntu-latest
+            python: 3.8
+            toxenv: pyinstaller
+
+          # Test against Python dev in cron job.
+          - name: Python dev with basic dependencies
+            skip: ${{ github.event_name != 'schedule'}}
+            python: 3.10-dev
+            toxenv: pydev-test
+
+    steps:
+    - name: Checkout code
+      if: ${{ skip != true }}
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Set up Python
+      if: ${{ skip != true }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python }}
+    - name: Install language-pack-de and tzdata
+      if: ${{ skip != true && matrix.os == 'ubuntu-latest' }}
+      run: sudo apt-get install language-pack-de tzdata
+    - name: Install graphviz
+      if: ${{ skip != true && matrix.toxenv == 'linkcheck' }}
+      run: sudo apt-get install graphviz
+    - name: Install Python dependencies
+      if: ${{ skip != true }}
+      run: python -m pip install --upgrade tox codecov
+    - name: Run tests
+      if: ${{ skip != true }}
+      run: |
+        echo "CC="$CC
+        echo "LC_CTYPE="$LC_CTYPE
+        echo "LC_ALL="$LC_ALL
+        tox ${{ matrix.toxargs}} -e ${{ matrix.toxenv}} -- ${{ matrix.toxposargs}}
```

Comment:
```suggestion
        tox ${{ matrix.toxargs }} -e ${{ matrix.toxenv }} -- ${{ matrix.toxposargs }}
```

Reply:
Sorry I just saw this and force pushed in the mean time. Can you just push a commit directly to fix this in both files? (there is a cron file now)